Shared/ar.lproj/Localizable.strings

```diff
@@ -1189,6 +1189,12 @@
 /* Setting to show Logins & Passwords quick access in the application menu */
 "Settings.ShowLoginsInAppMenu.Title" = "اعرض في قائمة التطبيق";
 
+/* Setting that appears in the Passwords screen to enable the built-in password manager so users can save their passwords. */
+"Settings.Passwords.SavePasswords.v103" = "احفظ كلمات السر";
+
+/* Title for the passwords screen. */
+"Settings.Passwords.Title.v103" = "كلمات السر";
+
```

Comment:
For translations, we import them from [Pontoon](https://pontoon.mozilla.org/) in our project. We import translations regularly in our project ([like this](https://github.com/mozilla-mobile/firefox-ios/pull/14881) for example). This means we can't add manually translations in the code, but instead need to go on Pontoon website to add them (example [here](https://pontoon.mozilla.org/ar/firefox-for-ios/firefox-ios.xliff/?string=220035)). Anyone can go in and suggest translations 🙏 

In the meantime, we'll need to remove this change since it would be overridden by the import otherwise.